plugins/package-managers/node/src/main/kotlin/PackageJson.kt

```diff
@@ -42,6 +42,26 @@ import org.ossreviewtoolkit.plugins.packagemanagers.node.PackageJson.Repository
 
 private val JSON = Json { ignoreUnknownKeys = true }
 
+/**
```

Comment:
The output which makes the deserializaton fail seems to be log output of "info" level.

I wonder if we can simply silence the info log output on stdout, by using `--silent` as option.
Have you considered this?

Reply:
> I wonder if we can simply silence the info log output on stdout, by using `--silent` as option.

Is this something you could still check, @oheger-bosch?

Reply:
Not easily, I am afraid. Unfortunately, the issue occurs only on our production system, and I was not able to reproduce it on a test system. It probably only occurs if there is a certain load.
While reducing the output may help in this case, it could make debugging harder in some scenarios.

Reply:
> While reducing the output may help in this case,

I guess the question is: Does `--slient` only *reduce* the output, or does it get rid of *all* otherwise unparsable output, so that no other change than just additionally passing `--slient` would be needed?

Reply:
I tested the following command:
`yarn info --json --silent commander@6.2.1`

The output is
```
{"type":"warning","data":"package.json: No license field"}
{"type":"inspect","data":{"name":"commander","description":"the complete solution ...
```

The cause of the warning is probably that I do not have any Yarn project locally. So, there are still multiple JSON objects in the output. This seems to be a standard behavior of Yarn. Also in the example from the [documentation about the info command](https://classic.yarnpkg.com/en/docs/cli/info#toc-yarn-info), the output has multiple lines. Maybe, to be on the safe side, we should look for the first object with type "inspect" ?

Reply:
> So, there are still multiple JSON objects in the output.

Thanks for checking.

> Maybe, to be on the safe side, we should look for the first object with type "inspect" ?

Makes sense to me!

Reply:
The implementation has been changed as discussed. Now the objects are parsed in a sequence, and the first one with type "inspect" is returned.